storage/aptosdb/src/state_store/mod.rs

```diff
@@ -148,20 +147,20 @@ impl StateStore {
         Ok(ret)
     }
 
-    pub fn get_account_count(&self, version: Version) -> Result<usize> {
+    pub fn get_leaf_count(&self, version: Version) -> Result<usize> {
```

Comment:
The State store is not explicitly a "tree", so maybe "get_value_count", or just "get_count'